### DISCUSSION TITLE
can not create issues under all repos in my account

### DISCUSSION THREAD

**Discussion by ismdeep** [community/community#37460] [Projects and Issues]
for example, you can not create new issue from here: https://github.com/ismdeep/macaron-demo/issues

**Comment by ossanna16** [comment]
Hi there @ismdeep and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by Bin4xin** [comment]
### Looks like an `ISSUE_TEMPLATE` BUG. 

`New issue` button link to https://github.com/ismdeep/macaron-demo/issues/new/choose

but this `macaron-demo` repo don not hava a ISSUE_TEMPLATE. So redirect to other repos which hava a ISSUE_TEMPLATE.

### solved

Just add `new` in url router by yourself:
 
`https://github.com/ismdeep/macaron-demo/issues/new` instead of click button.

hava a try? :D

<img width="1172" alt="截屏2022-10-29 23 46 51" src="https://user-images.githubusercontent.com/57884471/198840780-47063ff3-f88f-4ab3-ad42-57a93defa759.png">

### ref

see: https://docs.github.com/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms

**Comment by jfullstackdev** [comment]
and I can create a new issue without difficulty in one of my public repos

![image](https://user-images.githubusercontent.com/47092464/198876506-81246e8e-85c3-4f2e-b25d-58046c4a8705.png)

you need to fix that as was demonstrated above
